raiden/tests/integration/test_e2e.py

```diff
@@ -14,16 +21,27 @@ def test_fullnetwork(raiden_chain):
     asset_address = app0.raiden.chain.default_registry.asset_addresses()[0]
 
     amount = 80
+    random.seed(0)
     direct_transfer(app0, app1, asset_address, amount)
+    # Assert default identifier is generated correctly
+    fchannel = channel(app0, app1, asset_address)
+    last_transfer = get_sent_transfer(fchannel, 0)
+    random.seed(0)
```

Comment:
As discussed in person, it's in order to force the same "random" number to be generated inside the function and the test expecting function.